battle-service/src/main/java/com/company/battle/controllers/SettingsController.java

```diff
@@ -0,0 +1,37 @@
+package com.company.battle.controllers;
+
+import com.company.common.dtos.SettingsDto;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("api/v1/settings")
```

Comment:
Don't understand necessity of this controller. I think that settings should be part of game, and user should not be able to edit it independently